src/loader.js

```diff
@@ -191,6 +191,10 @@ function logloads(loads) {
       }
 
       load = createLoad(name);
+      if(!load.requests)
+        load.requests = [];
+
+      load.requests.unshift({as: request, from: refererName});
```

Comment:
Expanding on this: if there were a request tracking option, I'd think that systemjs-builder would set it by default, but systemjs wouldn't (or wouldn't in browser, for instance.) It's *very* useful during development, but I can see the concern about wasted memory/weedy values in production.